src/components/structures/ThreadPanel.tsx

```diff
@@ -221,6 +231,18 @@ const ThreadPanel: React.FC<IProps> = ({
                     setFilterOption={setFilterOption}
                     empty={threadCount === 0}
                 />}
+                footer={<>
+                    <BetaPill
+                        tooltipTitle={_t("Threads are a work in progress")}
+                        tooltipCaption={_t("Click to give feedback")}
+                        onClick={() => openFeedback?.()}
```

Comment:
```suggestion
                        onClick={openFeedback}
```

Otherwise the BetaPill always has clickable styling which isn't right